libshortfin/tests/apps/llm/components/tokenizer_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+
+
+@pytest.fixture
+def bert_tokenizer():
+    import shortfin_apps.llm.components.tokenizer as tokenizer
```

Comment:
I love that we have this :)

Reply:
xtensor has random number generators. just need to expose them to Python probably.